### ISSUE TITLE
Can you specify a custom mount path?

### ISSUE THREAD

**Issue by idodoron11** [piyushgarg/rclone-mount#14]
Is it possible to mount a remote storage under /storage/emulated/0/clouds/ ,  so that a non-root-privileged app would be able to access mounted storage? For instance, most music players only scan folders under /storage/emulated/0/ ,  so as long as the storage is mounted outside of this path's scope, a music app wouldn't be able to play remote music files.

**Comment by ioogithub** [comment]
Yes I need this as well. I want to automatically backup app settings and they all use the system file picker interface which cannot see outside of /storage/emulated/0. Has anyone else found a solution for this? I have been trying to make symlinks into /0 but nothing really works, android is complicated in how it maps this storage. In order for this app to be useful I need my apps to be able to see the rclone mounts.

**Comment by 1024mb** [comment]
Can't you use ``BINDSD=1`` in the params of your remote? That is supposed to bind the remote to a "Cloud" folder, just make sure no folder has that name already. Also you can specify folders to be synced with SDSYNCDIRS, all that info is on the Magisk module, I haven't tried any of that.

**Comment by idodoron11** [comment]
Where did you find this information? I don't quite understand where I should add these parameters and what they specify. I don't see anything related in here: [rclone-mount](https://github.com/piyushgarg/rclone-mount)...

**Comment by Geofferey** [comment]
This is not the official repo. 

This is:
https://github.com/Magisk-Modules-Repo/com.piyushgarg.rclone

A lil more info on custom params
https://github.com/Magisk-Modules-Repo/com.piyushgarg.rclone/issues/2
